rero_ils/es_templates/v6/record.json

```diff
@@ -1,33 +1,125 @@
 {
   "index_patterns": "*",
   "settings": {
-      "analysis": {
-          "tokenizer": {
-              "char_group_tokenizer": {
-                  "type": "char_group",
-                  "tokenize_on_chars": [
-                      "whitespace",
-                      "punctuation"
-                  ]
-              }
-          },
-          "analyzer": {
-              "global_lowercase_asciifolding": {
-                  "type": "custom",
-                  "tokenizer": "char_group_tokenizer",
-                  "filter": [
-                      "lowercase",
-                      "asciifolding",
-                      "length_min_3"
-                  ]
-              }
-          },
-          "filter": {
-              "length_min_3": {
-                  "type": "length",
-                  "min": 3
-              }
-          }
+    "analysis": {
+      "tokenizer": {
+        "char_group_tokenizer": {
+          "type": "char_group",
+          "tokenize_on_chars": [
+            "whitespace",
+            "punctuation"
+          ]
+        }
+      },
+      "analyzer": {
+        "english_analyzer": {
+          "type": "custom",
+          "tokenizer": "char_group_tokenizer",
+          "filter": [
+            "lowercase",
+            "english_stop",
+            "english_snowball",
+            "asciifolding",
+            "english_stemmer"
+          ]
+        },
+        "french_analyzer": {
```

Comment:
Please use the standard language analyzer.

Reply:
done